cmd/flux/get.go

```diff
@@ -33,3 +44,71 @@ func init() {
 		"list the requested object(s) across all namespaces")
 	rootCmd.AddCommand(getCmd)
 }
+
+type summarisable interface {
+	AsObject() runtime.Object
+	Len() int
+	SummariseAt(i int, includeNamespace bool) []string
```

Comment:
`SummariseItem` or `SummariseRow` maybe? The `At` did not really ring any bells with me.